src/utils/contracts/contractAddresses/internalAddresses.ts

```diff
@@ -60,6 +61,7 @@ export const internalAddressesPerNetwork: Record<
     wrappedNativeGateway: "0x0bb31c42D0692369Ba681A925C254fEB605c327b",
 
     explorer: "https://blockscout.com/xdai/mainnet",
+    rpcUrl: "https://rpc.xdaichain.com/",
```

Comment:
You added one for XDAI but not Rinkeby, despite adding Rinkeby above in `injectedConnectors`?

Reply:
Oops, I added it but I guess it got deleted on an undo I made or something

Reply:
I'll see if I can unify these in my changeset- having them in two separate files at once is messy and prone to making this same mistake again.